index.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$db = new PDO("mysql:host=db; dbname=collection_app", "root", "password");
+$db ->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+require 'functions.php';
+
+$query = getAllFromDB($db);
+$output = prepareOutput($query);
+
+?>
+
+<h2>NEW BOOK?</h2>
+
+<h2>BOOK COLLECTION</h2>
+
```

Comment:
Don't forget about the rest of the HTML you need to make this a valid HTML file. You also probably still want the responsive meta tag and a link to normalize!